apps/whale-api/src/module.indexer/model/dftx/pool.swap.ts

```diff
@@ -50,6 +51,11 @@ export class PoolSwapIndexer extends DfTxIndexer<PoolSwap> {
 
     for (const interval of AggregatedIntervals) {
       const previous = await this.aggregatedMapper.query(`${poolPairId}-${interval}`, 1)
+      if (previous.length === 0) {
+        // Logger Error instead of panic exiting to prevent cascade failure
+        this.logger.error(`Unable to find ${poolPairId}-${interval} for Aggregate Indexing`)
```

Comment:
```suggestion
        this.logger.error(`indexSwap ${transaction.txn.txid}: Unable to find ${poolPairId}-${interval} for Aggregate Indexing`)
```

Indicate the action to distinguish it from `invalidateSwap` error logging and the violating txid